packages/flutter/test/foundation/diagnostics_test.dart

```diff
@@ -1232,7 +1232,7 @@ void main() {
     expect(missing.isFiltered(DiagnosticLevel.info), isFalse);
     validateObjectFlagPropertyJsonSerialization(present);
     validateObjectFlagPropertyJsonSerialization(missing);
-  }, skip: isBrowser);
+  }, skip: isBrowser); // Reference needed
```

Comment:
@jonahwilliams is there an issue or comment I can add here?

Reply:
I think the closure serialization does not work the same on the web. I will take a look at this one tomorrow